pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyServiceStarter.java

```diff
@@ -84,15 +89,19 @@ public ProxyServiceStarter(String[] args) throws Exception {
             config.setZookeeperServers(zookeeperServers);
         }
 
-        if (!isBlank(globalZookeeperServers)) {
+        if (!isBlank(globalZookeeperServers) && isBlank(configurationStoreServers)) {
```

Comment:
is isBlank on configurationStoreServers isn't necessary here. If globalZooKeepers and configurationStoreServers are both set, configurationStoreServers will be used anyhow